source/grid/manifold_lib.cc

```diff
@@ -1000,7 +1000,7 @@ SphericalManifold<dim, spacedim>::get_new_point(
 
 template <>
 Point<3>
-SphericalManifold<1, 3>::get_new_point(
+SphericalManifold<1, 3>::my_get_new_point(
```

Comment:
I'm fine with the rename, but - why do these functions even exist? All of them (except the template base-case) just call `do_get_new_point()`. Could you check if they can be deleted (perhaps after adding an `if constexpr` check)?